Source/Core/DolphinWX/VideoConfigDiag.h

```diff
@@ -239,6 +239,18 @@ class VideoConfigDiag : public wxDialog
       progressive_scan_checkbox->Disable();
       render_to_main_checkbox->Disable();
     }
+
+    // Don't enable 'vertex rounding' at native
+    if (vconfig.iEFBScale == SCALE_1X)
+    {
+      vertex_rounding_checkbox->Enable(false);
+    }
+    else
+    {
+      vertex_rounding_checkbox->Enable(true);
+      vconfig.bVertexRounding = vertex_rounding_checkbox->GetValue();
```

Comment:
Yeah, I must have been testing incorrectly because I thought was needed go from an IR other than 1x -> 1x -> back to that IR.  But it is not.  Thank you for questioning it!